recipe/meta.yaml

```
@@ -58,6 +58,8 @@ requirements:
     - pytorch                                          # [cuda_compiler_version not in (undefined, 'None')]
     - pytorch =*={{ torch_proc_type }}*                # [cuda_compiler_version not in (undefined, 'None')]
   run:
+    - ninja
+    - pynvml
     - hjson-py
     # NOTE(hadim): coming at https://github.com/conda-forge/staged-recipes/pull/19098
     # - ninja-python
```

Removing `ninja` as runtime dependency. It's complicated... see https://github.com/conda-forge/staged-recipes/pull/19098

```suggestion
    - pynvml
    - hjson-py
    # NOTE(hadim): coming at https://github.com/conda-forge/staged-recipes/pull/19098
    # - ninja-python
```